### PR TITLE
ID Card Scanner Feature (W.I.P)

### PR DESCRIPTION
**Pull request by marcosaia1812**
Needs testing on iOS. I have included a copy of the updated plugin package and referenced it from the package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,13 +34,13 @@
...
     "cordova-plugin-ionic-keyboard": "^2.1.3",
     "cordova-plugin-ionic-webview": "^4.2.1",
-    "cordova-plugin-scanbot-sdk": "4.8.0",
     "cordova-plugin-splashscreen": "^5.0.2",
     "cordova-plugin-statusbar": "^2.4.2",
     "cordova-plugin-whitelist": "^1.3.3",
     "cordova-sqlite-storage": "^5.1.0",
     "core-js": "^2.5.4",
     "rxjs": "~6.3.3",
+    "cordova-plugin-scanbot-sdk": "4.8.1-beta2",
```
Comment (MrDyro) [inline package.json:43]: Can we change the version to 4.9.0 ? The last digit is for bugfixes, but ID-Card scanner is a new feature.

File: src/app/home/home.page.html [hunk 2]
```diff
@@ -34,6 +34,9 @@
       <ion-item (click)="startBatchBarcodeScanner()">
         <ion-label>Scan a Batch of Barcodes</ion-label>
       </ion-item>
+      <ion-item (click)="startIdCardScanner()">
+        <ion-label>Scan an ID Card</ion-label>
```
Comment (MrDyro) [inline src/app/home/home.page.html:38]: Please rename to German ID Card

**Review by MrDyro — CHANGES_REQUESTED** [review]

File: src/app/home/home.page.html (hunk 2 above)
Comment (MrDyro) [reply src/app/home/home.page.html:38]: Please move this menu item to the section `Data Detectors`

File: package.json (hunk 1 above)
Comment (marcosaia1812) [reply package.json:43]: @Nikituh Could we release the 4.9.0 beta so that I can update the plugin version?

Comment (Nikituh) [reply package.json:43]: I don't really want to make another beta release just for the version number fix. I'll make the official release 4.9.0 and also update documentation

**Comment by marcosaia1812** [comment]
First implementation of [EPIC-626](https://scanbot.atlassian.net/browse/EPIC-626) is done.

**Pull request merged**